src/generator.cpp

```diff
@@ -1046,15 +1046,15 @@ void Generator::GeneratorImpl::processNode(const XmlNodePtr &node,
             }
         } else {
             std::string modelName = entityName(owningModel(component));
-            ErrorPtr err = Error::create();
+            IssuePtr issue = Issue::create();
 
-            err->setDescription("Variable '" + variableName
-                                + "' in component '" + component->name()
-                                + "' of model '" + modelName
-                                + "' is referenced in an equation, but it is not defined anywhere.");
-            err->setKind(Error::Kind::GENERATOR);
+            issue->setDescription("Variable '" + variableName
+                                  + "' in component '" + component->name()
+                                  + "' of model '" + modelName
+                                  + "' is referenced in an equation, but it is not defined anywhere.");
+            issue->setCause(Issue::Cause::GENERATOR);
```

Comment:
isn't a set level required also? or is there a default level of error?

Reply:
The default level is currently `ERROR` so that all the existing behaviour stays the same.